source/docs/hardware-components/kit-and-hardware-guide/rev-robotics.rst

```diff
@@ -21,7 +21,6 @@ Notable components
 - REV also offers punch tubing for teams wanting a more permanent mounting system while having the flexibility of extrusion.
 - REV uses 5 mm hex steel :term:`shafts <Shaft>` and a :term:`ball-bearing <Ball Bearing>` motion support system. The shaft may be cut to length, which can be very useful for certain use cases. Many vendors have added compatibility to 5 mm hex :term:`shaft <Shaft>`.
 - REV offers 3 types of :term:`HD HEX motors <HD Hex Motor>`: spur gear, planetary, and UltraPlanetary motors. The UltraPlanetary has customizable planetary ring gear ratios for a very affordable price.
-- The Core Hex Motor may be useful in certain use cases such as space constrained mechanisms.
```

Comment:
It may be worthwhile explaining how the Core Hex has a hard time fitting into any use cases at all -- teams often put them on intakes for the packaging but the 125 RPM is obviously a bit of an L. 

It's unfortunate that the space savings it gives are almost never worth the tradeoff given by working in a 550-class, but it's how FTC exists currently.

Reply:
That is fair. Core hex being so cheap is the only thing keeping me from immediately deleting it, but we have plenty of 90 degree options now so I guess its time to finally retire the Core Hex.